bootstrap/frontend/src/main/scala/io/udash/bootstrap/form/UdashForm.scala

```diff
@@ -156,8 +160,9 @@ object UdashForm {
     inputGroup(inputId, validation)(labelContent)(FileInput(name, acceptMultipleFiles, selectedFiles)((id := inputId) +: input))
 
   /** Creates checkbox. */
-  def checkbox(validation: Option[Modifier] = None)(labelContent: Modifier*)(property: Property[Boolean], input: Modifier*): Modifier =
-    div(BootstrapStyles.Form.checkbox)(
+  def checkbox(validation: Option[Modifier] = None, inputId: ComponentId = UdashBootstrap.newId())
```

Comment:
These helper methods (`*input`, `checkbox` etc.) need better argument documentation with content and input varargs.